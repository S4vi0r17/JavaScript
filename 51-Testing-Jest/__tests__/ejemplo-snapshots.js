const client = {
    name: 'Ada',
    balance: 500,
    age: 26,
    type: 'premium'
};

describe('Testing al cliente', () => {

    test('Es Ada', () => {
        // expect(client.name).toBe('Ada');
        expect(client).toMatchSnapshot(); // Snapshot
    });

    test('El cliente es premium', () => {
        expect(client.type).toBe('premium');
    });

    test('No es otro cliente', () => {
        expect(client.name).not.toBe('Grace');
    });

    test('Tiene 500 de balance', () => {
        expect(client.balance).toBeGreaterThan(400);
    });

    test('Es menor de 30 aÃ±os', () => {
        expect(client.age).toBeLessThan(30);
    });
});