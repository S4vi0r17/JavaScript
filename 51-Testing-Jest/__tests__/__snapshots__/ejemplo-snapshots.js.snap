// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Ada 1`] = `
Object {
  "age": 26,
  "balance": 500,
  "name": "Ada",
  "type": "premium",
}
`;
